python-threatexchange/threatexchange/exchanges/signal_exchange_api.py

```diff
@@ -272,7 +272,7 @@ class SignalExchangeAPIWithSimpleUpdates(
         state.TFetchCheckpoint,
         state.TFetchedSignalMetadata,
         t.Tuple[str, str],
-        state.TFetchedSignalMetadata,
+        t.Any,
```

Comment:
@Dcallies I'm not sure on the typing here. All changes besides `t.Any` expanded to 10 more errors where `SignalExchangeAPIWithSimpleUpdates` is being used.

Reply:
Hmm, this is definitely not right here, the other option is to just # type: ignore for now

Reply:
I can't get mypy to understand the `# type: ignore` directive here. No matter where I put it (above, same line, below as part of class) it get a version of the following:
```
threatexchange/exchanges/signal_exchange_api.py:269: error: Variance of TypeVar "state.TFetchedSignalMetadata" incompatible with variance in parent type  [type-var]
threatexchange/exchanges/signal_exchange_api.py:277: error: Unused "type: ignore" comment
```

Reply:
Synced offline, as ignore was not working I will merge as t.Any but created high-pri issue https://github.com/facebook/ThreatExchange/issues/1255